File: MyMonitorHub.JavaAgent/src/main/java/mymonitorhub/agent/core/GroupPluginThread.java
package mymonitorhub.agent.core;

/**
 * Runs a GroupPlugin — identical to PluginThread with multi=true.
 * Mirrors C# GroupPluginThread.
 */
public class GroupPluginThread extends PluginThread {

    public GroupPluginThread(int index) {
        super(index, true);
    }
}

File: MyMonitorHub.Agent/MyMonitorHub.Agent.Core/Core/GroupPluginThread.cs
namespace MyMonitorHub.Agent.Core
{
    internal class GroupPluginThread : PluginThread
    {
        internal GroupPluginThread(int index) : base(index, true)
        {
            Plugin!.HibernateSeconds = 30; // override the default of 2 minutes
        }
    }

    // class
}